cleanlab/datalab/internal/issue_manager/data_valuation.py

```diff
@@ -55,7 +76,7 @@ class DataValuationIssueManager(IssueManager):
         3: ["average_data_valuation"],
     }
 
-    DEFAULT_THRESHOLDS = 1e-6
+    DEFAULT_THRESHOLD = 1e-6
```

Comment:
Consider if default of 0.5 makes more sense here. 

If the original values are in [-1, 1] and mapped to [0, 1] via 0.5*x + 1 transform, then threshold of 0.5 would flag datapoints with negative original value (which seems natural to me)

Reply:
Yes it makes sense, updated.